src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
+    private boolean isActive = false;
+
+    public abstract void doWork();
+
+    public abstract void stopWork();
+
+    public boolean isActive() {
```

Comment:
how and where do we use this method?

Reply:
Well, it might be used to check whether any machine is doing something. I have added it to the Main class.